packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -563,16 +564,23 @@ class PageTransitionsTheme with Diagnosticable {
   /// Constructs an object that selects a transition based on the platform.
   ///
   /// By default the list of builders is: [FadeUpwardsPageTransitionsBuilder]
-  /// for [TargetPlatform.android], and [CupertinoPageTransitionsBuilder] for
-  /// [TargetPlatform.iOS] and [TargetPlatform.macOS].
-  const PageTransitionsTheme({ Map<TargetPlatform, PageTransitionsBuilder> builders = _defaultBuilders }) : _builders = builders;
+  /// for [TargetPlatform.android] and [TargetPlatform.fuchsia],
+  /// [CupertinoPageTransitionsBuilder] for [TargetPlatform.iOS], and no
+  /// animated transition for other platforms or if the app is running on the
+  /// web.
+  const PageTransitionsTheme({
+    Map<TargetPlatform, PageTransitionsBuilder> builders = kIsWeb ? _defaultWebBuilders : _defaultBuilders,
```

Comment:
`kIsWeb` is true both in desktop and mobile browsers. Do we want to shut off transitions in mobile browsers too?

Reply:
I believe we do. That is the feedback I got when I asked this question on this [issue](https://github.com/flutter/flutter/issues/82053#issuecomment-841457312). For most web apps that aren't PWAs I would think they would want consistent behavior. For apps that do want transitions, they can provide them either manually or by using the theme settings detailed above.